packages/odyssey-react/src/components/Tab/Tab.test.tsx

```diff
@@ -81,6 +81,76 @@ describe("Tabs", () => {
     expect(handleTabChange).toHaveBeenCalledTimes(1);
   });
 
+  it('should focus the last element when the end key is pressed', async () => {
+    const { getByText, getByRole } = render(
+      <Tabs id={id} selectedId="sb-tabs-example-2" ariaLabel={ariaLabel}>
+        <Tabs.Panel id="sb-tabs-example-1" label="Tab 1">TabPanel 1</Tabs.Panel>
+        <Tabs.Panel id="sb-tabs-example-2" label="Tab 2">TabPanel 2</Tabs.Panel>
+        <Tabs.Panel id="sb-tabs-example-3" label="Tab 3">TabPanel 3</Tabs.Panel>
+      </Tabs>
+    );
+
+    fireEvent.keyUp(getByRole('tablist'), {key: 'Tab', code: 'Tab'})
+    fireEvent.keyUp(getByRole('tablist'), {key: 'End', code: 'End'})
+
+    await waitFor(() => {
+      expect(getByText('Tab 3')).toHaveFocus()
+    })
+  });
+
+  it('should focus the last element when the home key is pressed', async () => {
```

Comment:
Do `home` and `end` have the same behavior?

Reply:
Nope!

![CleanShot 2021-09-16 at 10 27 30](https://user-images.githubusercontent.com/62259644/133657772-21bee5f1-fc71-4203-ae83-5cf32d2aa81e.png)